docs/user-guide/installation.rst

```diff
@@ -9,41 +9,184 @@ Installation
 
 .. edit-link::
 
-The recommended way to get started with Rasa is via ``pip``:
+Quick Installation
+~~~~~~~~~~~~~~~~~~
 
-.. copyable::
+You can get started using ``pip`` with the following command.
 
-    pip install rasa-x --extra-index-url https://pypi.rasa.com/simple
+.. code-block:: bash
 
-This will install both Rasa and Rasa X.
-If you don't want to use Rasa X, run ``pip install rasa`` instead.
+    $ pip install rasa-x --extra-index-url https://pypi.rasa.com/simple
 
-.. raw:: html
+.. note::
+
+    You'll need a minimum Python version of 3.5.4 to install Rasa.
+
+This will install both Rasa and Rasa X on your system.
+
+Once you're done with this, you can head over to the tutorial!
+
+.. button::
+   :text: Next Step: Tutorial
+   :link: ../rasa-tutorial/
+
+**For a more detailed guide on setting up, follow along the** :ref:`installation guide <installation_guide>`.
 
-     Unless you've already got numpy & scipy installed, we highly recommend
-     that you install and use
-     <a class="reference external" href="https://www.anaconda.com/download/"
-     target="_blank">Anaconda</a>.
 
-If you want to use the development version of Rasa, you can get it from GitHub:
+
```

Comment:
@erohmensing Ideally, I'd want to add the contents directive here (with a local thing below it), but it just disappears if I do it at this particular location. Any ideas?